images/icons.svg

```diff
@@ -0,0 +1,1140 @@
+<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><symbol id="add-cell" viewBox="0 0 512 512">	<path class="lexicon-icon-body" fill="none" d="M455.1,0H56.9C25.5,0,0,25.5,0,56.9v398.2C0,486.5,25.5,512,56.9,512h398.2c31.4,0,56.9-25.5,56.9-56.9V56.9C512,25.5,486.5,0,455.1,0z"></path>
```

Comment:
Ideally we wouldn't have to copy/paste this in, but I think this is fine for now until we find a way to serve it statically from our node_modules.